runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -1508,7 +1578,7 @@ public void translate(StreamingPubsubIOWrite transform, TranslationContext conte
       // Using a GlobalWindowCoder as a place holder because GlobalWindowCoder is known coder.
       stepContext.addEncodingInput(
           WindowedValue.getFullCoder(VoidCoder.of(), GlobalWindow.Coder.INSTANCE));
-      stepContext.addInput(PropertyNames.PARALLEL_INPUT, context.getInput(transform));
+      stepContext.addInput(PropertyNames.PARALLEL_INPUT, input);
```

Comment:
BTW doesn't Java work (for runner v2) when we just set PropertyNames.PUBSUB_SERIALIZED_ATTRIBUTES_FN to empty string ?

Reply:
Please refer to L1574. In java, it's not an empty string but a serialized func, though the func is not used in portable run.